docker-compose.yml

```diff
@@ -45,6 +45,8 @@ services:
 
   web:
     restart: unless-stopped
+    ipc: host
+    shm_size: '4gb'
```

Comment:
Is this needed ?

Reply:
It took quite a few resources for this to run in a container. I can attempt to run them without this.